contracts/libs/decimals/DecimalsConverter.sol

```diff
@@ -104,6 +104,41 @@ library DecimalsConverter {
         return convertSafe(amount_, decimals_, round18);
     }
 
+    /**
+     * @notice The function to do the token precision convertion
+     * @param amount_ the amount to convert
+     * @param baseToken_ current token
+     * @param destToken_ desired token
+     * @return the converted number
+     */
+    function roundTokens(
```

Comment:
Could you please explain why would one need this function?

Reply:
Well, I had a case in a previous project where I needed to compare the amount of stable coins from Curve, and I had to adjust one amount to match the decimal precision of the second amount. So, when I wrote this function, I was thinking about that particular case. 😄